depends/patches/qt/mac-qmake.conf

```
@@ -18,5 +18,6 @@ QMAKE_MAC_SDK.macosx.PlatformPath = /phony
 !host_build: QMAKE_CXXFLAGS += -target $${MAC_TARGET}
 !host_build: QMAKE_LFLAGS += -target $${MAC_TARGET}
 QMAKE_AR = $${CROSS_COMPILE}ar cq
+QMAKE_AR_LTCG = llvm-ar cqs
```

Shouldn't this be an ar from our binutils? Otherwise isn't this requiring llvm to be installed?

---

Tested this branch, without llvm (which shouldn't become a dependency), this will fail with:
```bash
make[4]: llvm-ar: No such file or directory
```

Any reason we can't do?
```suggestion
QMAKE_AR_LTCG = $${CROSS_COMPILE}ar cqs
```

---

It does not work.